src/background/chrome-api.ts

```diff
@@ -239,7 +242,10 @@ export async function executeFunction<Args extends unknown[], Result>(
   }
 
   const code = codeStringForFunctionCall(func, args);
-  const result = await chromeAPI_.tabs.executeScript(tabId, { frameId, code });
+  const result = (await chromeAPI_.tabs.executeScript(tabId, {
+    frameId,
+    code,
+  })) as Result[];
```

Comment:
Nor putting a lot of effort into this, as this is part of Manifest v2 and getting removed soon.